src/views/DatasetManager/components/DatasetMetadata/DatasetMetadata.js

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Card, Grid, IconButton, Paper } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import FileCopyOutlinedIcon from '@mui/icons-material/FileCopyOutlined';
+import { TagsEditor } from '@cosmotech/ui';
+import { useDatasetMetadata } from './DatasetMetadataHook';
+import { DescriptionEditor, MetadataItem } from './components';
+import { ApiUtils } from '../../../../utils/ApiUtils.js';
```

Comment:
we can import it directly from _/utils_ :slightly_smiling_face:

Reply:
Nice catch, thanks :smile: